docs/dev/perl5/prep/inputs/5.16.2.source.txt

```diff
@@ -0,0 +1,40 @@
+The Perl 5 development team is gratified to announce the release of
```

Comment:
Can we find a different place to put the prep content?  Right now it's in the *serving* tree and it's kind of clutter.

Maybe a new git repository?  It doesn't have to be anywhere near this one.  I could even make a good argument for keeping it in the main perl5 repo.

I had thought we had built something into Combust for private directories, but I can't find it.  If you really want to keep them in this repo, then they should be in the same directory and something we're ok with serving.